static/app/views/replays/replayTable/index.tsx

```diff
@@ -215,4 +217,22 @@ const StyledAlert = styled(Alert)`
   margin-bottom: 0;
 `;
 
-export default ReplayTable;
+type AdditionalProps =
+  | {
+      group: Group;
+      showReplayPlayer: true;
+    }
+  | {
+      group?: undefined;
+      showReplayPlayer?: false;
+    };
```

Comment:
@michellewzhang Similar to before. I want to make sure that if someone sets `showReplayPlayer` to true, that a group is passed in because otherwise things will break. But if `showReplayPlayer` isn't true, we don't need the group info.